kubetest2-tf/hack/terraform_install.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Copyright 2025 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TF_VERSION="v1.9.8"
+TERRAFORM_PROVIDER_IBM_VERSION="v1.73.0"
+TERRAFORM_PROVIDER_NULL_VERSION="v3.2.3"
+TF_PLUGIN_PATH=/root/.terraform.d/plugins/registry.terraform.io
+
+install_terraform_power(){
+    if [ -f "/go/bin/terraform" ]; then
+        echo "terraform already present"
+    else
+        cd /tmp
+        git clone -b $TF_VERSION --single-branch https://github.com/hashicorp/terraform.git
```

Comment:
use curl to pull the tagged version from the git repo

Reply:
this applies to other plugins as well.

Reply:
Fixed.